src/components/inputs/input-phone/input-phone.types.ts

```diff
@@ -1,12 +1,12 @@
-import { CountryCode, ParseError, PhoneNumber } from 'libphonenumber-js';
+import { CountryCode, PhoneNumber } from 'libphonenumber-js';
 
 import { InputTextProps } from '../input-text/types';
 
 export type InputPhoneProps = InputTextProps & {
   countryCode?: CountryCode;
   onCountryPickerPress?: () => void;
   onChangePhoneNumber?: (phoneNumber: PhoneNumber) => void;
-  onError?: (error: ParseError) => void;
+  onValidChange?: (valid: boolean) => void;
```

Comment:
`isValid` to have consistent variable naming for Boolean

Reply:
What does this method represents? A change in validity or called every time we check the validity of a value?

Reply:
for now it is call everytime the value change